### ISSUE TITLE
Can you provide the processed data or the related processing code?

### ISSUE THREAD

**Issue by Haoqing-Wang** [sstzal/DiffTalk#7]
Nice job! Can you released the used dataset?

**Comment by DavidKong96** [comment]
hope to know which  face det model used..

**Comment by lmpeng12** [comment]
+1

**Comment by MengShen0709** [comment]
+1

**Comment by MengShen0709** [comment]
@DavidKong96 
I guess authors might use dlib to extract the landmarks

The partial landmarks are defined in their [dataloader](https://github.com/sstzal/DiffTalk/blob/d34480f24408827c7535cd4b653bb1ebdba981b2/ldm/data/talk_data_ref_smooth.py#L65):
```
        landmarks_img = landmarks[13:48]
        landmarks_img2 = landmarks[0:4]
        landmarks_img = np.concatenate((landmarks_img2, landmarks_img))
```
I use dlib to extracts 68 points, and follow the authors to only keep some of them. It turns out, by doing so, we could remove bottom half of landmarks. This landmark masking is in accordance with the image masking for the model to learn how to generate lip movement.

Here is an illustration:
![landmark](https://github.com/sstzal/DiffTalk/assets/15813570/f2628df6-3809-4dc0-b735-289b4382c6be)

But I haven't reproduced the results, so it is just my guess.

Feel free to discuss.

**Comment by Haoqing-Wang** [comment]
before obtain landmark, we need to detect the facial RoI advance. But when the model can not detete the face, how to obtain the landmark? we use dlib to obtain landmark.

> detector = dlib.get_frontal_face_detector()
predictor = dlib.shape_predictor("shape_predictor_68_face_landmarks.dat")
gray = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
rect = detector(gray)[0]
shape = predictor(gray, rect)

rect could be None.

**Comment by jianmanLin** [comment]
> @DavidKong96 I guess authors might use dlib to extract the landmarks
> 
> The partial landmarks are defined in their [dataloader](https://github.com/sstzal/DiffTalk/blob/d34480f24408827c7535cd4b653bb1ebdba981b2/ldm/data/talk_data_ref_smooth.py#L65):
> 
> ```
>         landmarks_img = landmarks[13:48]
>         landmarks_img2 = landmarks[0:4]
>         landmarks_img = np.concatenate((landmarks_img2, landmarks_img))
> ```
> 
> I use dlib to extracts 68 points, and follow the authors to only keep some of them. It turns out, by doing so, we could remove bottom half of landmarks. This landmark masking is in accordance with the image masking for the model to learn how to generate lip movement.
> 
> Here is an illustration: ![landmark](https://user-images.githubusercontent.com/15813570/254321056-f2628df6-3809-4dc0-b735-289b4382c6be.png)
> 
> But I haven't reproduced the results, so it is just my guess.
> 
> Feel free to discuss.

Hello, did you successfully reproduce this paper? As a result of my training, the inpainting area will keep shaking, and then the training loss will drop rapidly at the beginning, and then it will shake within a small area. I feel very happy troubled

**Comment by Utkarsh-shift** [comment]
same request
